packages/usa-in-page-navigation/src/index.js

```diff
@@ -216,9 +248,11 @@ const createInPageNav = (inPageNavEl) => {
     const anchorTag = document.createElement("a");
     const textContentOfLink = el.textContent;
     const tag = el.tagName.toLowerCase();
+    const topHeadingLevel = getTopLevelHeading(sectionHeadings);
```

Comment:
Just noting this runs when there's only a single heading passed. This seems okay for the small amount of headers available.